File: learning/junior-training-sheet/taba/gregs_workout.py
def main():
    n = int(input().strip())
    es = list(map(int, input().split()))

    biceps = 0
    chest = 0
    back = 0
    for i in range(len(es)):
        if i % 3 == 0:
            chest += es[i]
        elif i % 3 == 1:
            biceps += es[i]
        elif i % 3 == 2:
            back += es[i]

    if biceps >= chest and biceps >= back:
        print("biceps")
    elif chest >= biceps and chest >= back:
        print("chest")
    else:
        print("back")


if __name__ == "__main__":
    main()

File: gym/junior-training-sheet/taba/gregs_workout.cpp
#include <iostream>

using namespace std;

int main() {
  int n, m;
  int chest = 0, biceps = 0, back = 0;

  cin >> n;
  for (int i = 0; i < n; i++) {
    cin >> m;
    if (i % 3 == 0) {
      chest += m;
    } else if (i % 3 == 1) {
      biceps += m;
    } else {
      back += m;
    }
  }

  if (chest > biceps) {
    if (chest > back) {
      cout << "chest\n";
    } else {
      cout << "back\n";
    }
  } else if (biceps > back) {
    cout << "biceps\n";
  } else {
    cout << "back\n";
  }

  return 0;
}